.husky/pre-commit

```diff
@@ -2,3 +2,4 @@
 . "$(dirname "$0")/_/husky.sh"
 
 npx --no-install lint-staged
+yarn betterer precommit
```

Comment:
How expensive is this command? (in term of time of execution). If it's too long I fear it's going to just have people do the `commit -n` and cancel all checks

Reply:
It's about 16s for me (m1 mac), It's a lot of files to check. Is it too long? 🤔

Reply:
Oh yeah, that's a bit long for a precommit... :grimacing: